lib/Alchemy/Phrasea/Controller/Prod/StoryController.php

```diff
@@ -71,6 +73,10 @@ public function postCreateFormAction(Request $request)
         }
 
         $recordAdapter = $story->set_metadatas($metadatas);
+        // order to write meta in file
+        $this->dispatch(WorkerEvents::RECORDS_WRITE_META,
+            new RecordsWriteMetaEvent([$story->getRecordId()], $story->getDataboxId()));
+
         // tell phraseanet to rebuild subdef
         $this->dispatch(RecordEvents::SUBDEFINITION_CREATE, new SubdefinitionCreateEvent($recordAdapter));
 
```

Comment:
to be verified : we may not need to create subdefs for a story, since it has no 'document'. Check that the "folder" thumbnail / preview are still displayed.